Sources/VaporToolbox/Version.swift

```diff
@@ -22,7 +22,10 @@ public final class Version: Command {
         console.success("\(version)")
 
         guard verifySwiftProject() else { return }
-        guard try verifyVaporProject() else { return }
+        guard try isVaporProject(with: console) else {
+            console.warning("No Vapor dependency detected, unable to log Framework Version")
```

Comment:
The line above check if it's a swift project, if it's not, then no log. This only prints f it's a swift project and Vapor isn't detected there.